src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
This approach is correct according to mentor guidelines. 
However in real life, in order to provide a globally accessible storage service with dynamically changing internal state, you should follow the Singleton pattern:

1. Make constructors private - no one should be able to instantiate the singleton.
2. Store the only class instance internally as the private static field (T instance). The state can be maintained as a non-static private field within the instance - provide necessary accessors and mutators to singleton class (e.g. T.addFruit(...).
3. Create public static T getInstance() method that provides he internally stored instance:
```java
public static T getInstance() {
  if(instance == null) {
    instance = new T();
  }
  return instance;
}
```